MVP.md

```diff
@@ -11,3 +12,27 @@ In all of our investigations to-date, we have discovered one protocol which has
 
 The best solution we have found is a provider known as [Unlock Protocol](https://unlock-protocol.com/) which offers ERC-721 tokens that are fully configurable and can be implemented on a variety of common L1 and L2 blockchains.  From a pure functional perspective
 
+</details>
+
+# Problem = Signal 2 Noise Ratio
+
+<details><summary>The tokenomics of LexDAO are simple but the interactions are subtle:</summary>
+1. mmmber - annual fixed subscriptionin fiat but payable in stablecoins;
+   - working for membership - a loosely pegged conversion from base membership to LXCHAT, a timebank
+   - this timebank represents ~15 hrs of sweat as in-kind contribution in lieu of the annual membership subscription
+   - it is arguable whether these l'externs should have access to all guild channels given the semi-confidential nature of some projects but conceptually they have all access to the same platforms members have (discord, hackMD, github, etc)
```

Comment:
I'd like to move this out from a Membership oriented topic to a topic related to roles and priviledges